src/components/LoginPage/Slide3.js

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import arrow from 'components/LoginPage/images/arrow.svg'
+import slideImage1 from 'components/LoginPage/images/slide-3_img-1.svg';
+import slideImage2 from 'components/LoginPage/images/slide-3_img-2.svg';
+
+const Slide = styled.section`
+  padding-bottom: 0%;
+  text-align: center;
+  .note-example {
+    width: 61%;
+  }
+   .edit-example {
+    position: relative;
+    margin-top: -9%;
+  }
+  .bottomPosition {
```

Comment:
What is a "bottom position" ? Doesn't make any sense to me.

Reply:
New variant in commit [661b2d7](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/661b2d774c1c591c5624f53d6015c4d90343adc1)